src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-unused-vars */
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  let result = 0;
```

Comment:
- [ ] Should this function return the result? It sets the result but returns an undefined value. 
We could also get rid of the result variable, and return the result directly within the if-else body.